tests/unit/s2n_tls13_keys_test.c

```diff
@@ -238,6 +250,25 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(s2n_tls13_derive_application_secret(&secrets, &hash_state, &server_application_secret, S2N_SERVER));
     S2N_BLOB_EXPECT_EQUAL(expect_derived_server_application_traffic_secret, server_application_secret);
 
+    /* Update handshake hashes with Client Finished */
+    EXPECT_SUCCESS(s2n_hash_update(&hash_state, client_finished.data, client_finished.size));
+    
+    /* Test session resumption secret */
+    s2n_tls13_key_blob(master_resumption_secret, secrets.size);
+    EXPECT_SUCCESS(s2n_tls13_derive_resumption_master_secret(&secrets, &hash_state, &master_resumption_secret));
+
+    S2N_BLOB_EXPECT_EQUAL(expect_derived_master_resumption_secret, master_resumption_secret);
+
+    /* Test individual session resumption ticket secret */
+    s2n_tls13_key_blob(session_ticket_secret, secrets.size);
+    uint8_t nonce[2] = {0};
+    struct s2n_blob ticket_nonce = {0};
+    EXPECT_SUCCESS(s2n_blob_init(&ticket_nonce, nonce, sizeof(nonce)));
```

Comment:
You probably want to treat this like any other hex string from the test vector RFC (use S2N_BLOB_FROM_HEX and list it with the others).